From a028ee074ae0f45f278e751dda1a4653042c4387 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Wed, 7 Nov 2018 16:42:09 +0800
Subject: [PATCH] Revert "ARM: dts: rockchip: remove the psci on rk3036 kylin"

This reverts commit e0145529893b9ff57011f4cdbdc46ad029091e5b.

Now we can support the psci, let's enable it.

Change-Id: I39723b8ca2e26487d3a59a52469a515343bcdf6d
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm/boot/dts/rk3036-kylin.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3036-kylin.dts b/arch/arm/boot/dts/rk3036-kylin.dts
index ce28543897c7..05415472859d 100644
--- a/arch/arm/boot/dts/rk3036-kylin.dts
+++ b/arch/arm/boot/dts/rk3036-kylin.dts
@@ -62,8 +62,6 @@
 		};
 	};
 
-	/delete-node/ psci;
-
 	sdio_pwrseq: sdio-pwrseq {
 		compatible = "mmc-pwrseq-simple";
 		pinctrl-names = "default";
-- 
2.35.3

